src/main/java/org/folio/rest/client/impl/FeesFinesModuleClientImpl.java

```diff
@@ -40,9 +43,9 @@ public Future<Integer> getOpenAccountsCountByUserId(String userId, OkapiConnecti
   }
 
   @Override
-  public Future<Integer> getManualBlocksCountByUserId(String userId, OkapiConnectionParams connectionParams) {
-    String query = StringUtil.urlEncode("(userId==" + StringUtil.cqlEncode(userId) + ")");
-    String requestUrl = connectionParams.getOkapiUrl() + "/manualblocks?limit=0&query=" + query;
+  public Future<Integer> getNonExpiredManualBlocksCountByUserId(String userId, OkapiConnectionParams connectionParams) {
+    String query = StringUtil.urlEncode("(userId==" + StringUtil.cqlEncode(userId) + " AND expirationDate>=" + LocalDate.now() + ")");
```

Comment:
Offset datetime? Probably it will be a really rare case that the timezone of mod-users-bl is not the same as the timezone of mod-users